### ISSUE TITLE
Black screen with Japanese PS2

### ISSUE THREAD

**Issue by GT2000byte** [CTurt/FreeDVDBoot#127]
Hello, I've been trying to use FreeDVDBoot on a Japanese PS2, however the DVD just doesn't start at all no matter if I removed my controller, unplugged my Memory Cards, reseted the console several times. Also before anyone asks, my PS2 actually plays DVD's as well as games normal,  the console is set in English and the DVD used for FreeDVDBoot is a DVD-R.

The version information:

Console: SCPH-70000
Browser: 1.40
CD Player: 2.00
PlayStation Driver: 1.02 (It's options are in ''Standard'')
DVD Player: 3.10

Please let me know what I did wrong.

**Comment by SoraBlack** [comment]
Do you have your console on english? (Language I mean...)

**Comment by GT2000byte** [comment]
> Do you have your console on english? (Language I mean...)

Yes I have. In fact my console has been in English since I got it.

**Comment by CTurt** [comment]
Hey, I'm really sorry. I think I broke the imge recently, I reverted it back. Please download the image again, and try again. Hoping it fixes the issue! Thanks!

**Comment by GT2000byte** [comment]
> Hey, I'm really sorry. I think I broke the imge recently, I reverted it back. Please download the image again, and try again. Hoping it fixes the issue! Thanks!

Seems about right lol
I have downloaded the new image so I'll check it later.
